setpass/templates/helpdesk_ticket.txt

```diff
@@ -0,0 +1,7 @@
+[{username}]
```

Comment:
I agree, I wasn't really sure where it should go.  Maybe create a different subfolder?  Or maybe define it as a config value.